### ISSUE TITLE
Issue with "contains" filter and Array

### ISSUE THREAD

**Issue by pepe95270** [APY/APYDataGridBundle#872]
Hi all,
I think there is an issue with the "contains" filter when using an Array.
For exemple, if my array is (in my database):      `a:3:{i:0;s:6:"France";i:1;s:5:"Congo";i:2;s:6:"Italie"}`
In the grid it will show: 
> France
> Congo
> Italie

But if I try to filter this column with: `Contains: "Fra"` in the grid it will return "No result" ! :(
It's strange because if I try to filter this column with: `Contains: "France"` it will return my entry this time.

To sum up, the "contains" filter is the same as "equal to" filter on an array...

Any idea why?

Thank you :)
